frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -46,7 +45,7 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample }) => {
     const isReachable = gitService && (await gitService.isRepoReachable());
     setFieldValue('git.isUrlValidating', false);
     if (isReachable) {
-      setFieldValue('git.isUrlValidated', true);
+      setFieldValue('git.isUrlValidated', ValidatedOptions.success);
```

Comment:
Maybe rename this from `isUrlValidated` to something else since its not a boolean anymore. Something like `validationStatus`?